tests/test_coaddInputs.py

```diff
@@ -107,6 +107,7 @@ def makeExposure(self, universalId):
         expInfo.setPhotoCalib(lsst.afw.image.makePhotoCalibFromCalibZeroPoint(1.1e12, 2.2e10))
         expInfo.setApCorrMap(self.makeApCorrMap())
         expInfo.setValidPolygon(lsst.afw.geom.Polygon(lsst.geom.Box2D(bbox).getCorners()))
+        expInfo.setFilterLabel(lsst.afw.image.FilterLabel(physical="fakeFilter", band="fake"))
```

Comment:
Is this needed, given that the original test didn't set a `Filter`?

Reply:
Yes: ExposureInfo always had a default Filter before (it was the one non-shared_ptr in there), so the code was getting by with a default constructed Filter until now.